tools/shared/src/main/scala/org/scalajs/core/tools/linker/frontend/GenIncOptimizer.scala

```diff
@@ -7,7 +7,7 @@
 \*                                                                      */
 
 
-package org.scalajs.core.tools.optimizer
+package org.scalajs.core.tools.linker.frontend
```

Comment:
I suggest sending `GenIncOptimizer`, `IncOptimizer`, `ParIncOptimizer` and `OptimizerCore` to a subpackage `frontend.optimizer`.